package.json

```diff
@@ -35,28 +36,29 @@
   "author": "Ernesto Garcia <gnapse@gmail.com> (http://gnapse.github.io/)",
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "^7.5.1",
+    "@babel/runtime": "^7.8.3",
     "@types/testing-library__jest-dom": "^5.0.0",
-    "chalk": "^2.4.1",
-    "css": "^2.2.3",
+    "chalk": "^3.0.0",
+    "css": "^2.2.4",
     "css.escape": "^1.5.1",
-    "jest-diff": "^24.0.0",
-    "jest-matcher-utils": "^24.0.0",
-    "lodash": "^4.17.11",
-    "pretty-format": "^24.0.0",
+    "jest-diff": "^25.1.0",
+    "jest-matcher-utils": "^25.1.0",
+    "lodash": "^4.17.15",
+    "pretty-format": "^25.1.0",
     "redent": "^3.0.0"
   },
   "devDependencies": {
-    "jest-watch-select-projects": "^0.1.2",
-    "jsdom": "^15.1.0",
-    "kcd-scripts": "^1.4.0"
+    "jest-environment-jsdom-sixteen": "^1.0.0",
+    "jest-watch-select-projects": "^1.0.0",
+    "kcd-scripts": "^4.1.0"
   },
   "eslintConfig": {
     "extends": "./node_modules/kcd-scripts/eslint.js",
     "rules": {
       "babel/no-invalid-this": "off",
-      "import/prefer-default-export": "off",
-      "import/no-unassigned-import": "off"
+      "import/no-extraneous-dependencies": "off",
```

Comment:
Why is this rule complaining all of the sudden?

Reply:
It's because of an update in `kcd-scripts`' ESLint config

Reply:
So it wasn't there before? What is it complaining about?

Reply:
It's complaining about `jsdom` not in our dependencies.

Reply:
Maybe we should just ignore that one in `__tests__`? 🤔

Reply:
`jsdom` used to be in `devDependencies` but it got [removed](https://github.com/testing-library/jest-dom/pull/190/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L51). Rather than disabling the rule, could the dependency be restored?

Reply:
@jgoz It's included by `jest-environment-jsdom-sixteen`

Reply:
Right, but there is a `require("jsdom")` statement inside this repository, which technically makes it an explicit (dev) dependency of this project. Disabling the rule as a whole will prevent us from detecting when _any_ transitive dependency is imported as though it were a direct dependency, not just this one instance.